metrics/network/cpu_statistics_iperf.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'iperf3' network benchmark and
+# we retrieve cpu information while running the networking
+# test with 'perf'.
```

Comment:
can we add a note here explaining why we are measuring the CPU info - noting that (we are hoping ;-) that cpu utilisation is a more stable value to measure to detect changes than actual throughput. Also, discuss how we are not running a 'flat out' test here, as othewise we fully expect that to hit 100% utilisation on a local pure-software network stack.